rasa/shared/core/domain.py

```
@@ -1197,14 +1198,9 @@ def is_domain_file(filename: Text) -> bool:
 
         if not is_likely_yaml_file(filename):
             return False
-        try:
-            content = rasa.shared.utils.io.read_yaml_file(filename)
-            if any(key in content for key in ALL_DOMAIN_KEYS):
-                return True
-        except YAMLError:
```

this function won't swallow errors anymore